src/app.tsx

```diff
@@ -11,6 +11,10 @@ import { SacnRendererProvider } from './contexts/SacnRendererContext';
 import { SerialProvider } from './contexts/SerialContext';
 import { ShortcutProvider } from './contexts/ShortcutContext';
 import { WledRendererProvider } from './contexts/WledRendererContext';
+import { initRenderListeners } from './system_interfaces/engine';
+
+// Initialize Tauri render event listeners
+initRenderListeners();
```

Comment:
Can we call this directly in `engine.ts` instead of up here? Are there any downsides to doing so?